scripts/cnode-helper-scripts/cncli.sh

```diff
@@ -797,10 +805,187 @@ cncliPTsendslots() {
   done
 }
 
+#################################
+# epochdata table load process  #
+#################################
+
+getCurrNextEpoch() {
+  getNodeMetrics
+  curr_epoch=${epochnum}
+  next_epoch=$((curr_epoch+1))
+}
+
+runCurrentEpoch() {
+  getKoiosData
+  getCurrNextEpoch
+  echo "Processing current epoch: ${curr_epoch}"
+  stake_param_curr="--active-stake ${active_stake_set} --pool-stake ${pool_stake_set}"
+
+  ${CNCLI} leaderlog ${cncliParams} --consensus "${consensus}" --epoch="${curr_epoch}" --ledger-set current ${stake_param_curr} |
```

Comment:
According to cncli documentation:
> --epoch <epoch>
>             Provide a specific epoch number to calculate for and ignore --ledger-set option
So either skip `--epoch` parameter or `--ledger-set`. I dont think it matter that --ledger-set is present as --epoch will override it but makes it clearer if one or the other is skipped.